File: iniciante/1012/b1012.java
import java.util.Locale;
import java.util.Scanner;

public class b1012 {
  public static void main(String[] args) {
    Locale.setDefault(Locale.US);
    Scanner sc = new Scanner(System.in);

    double A = sc.nextDouble();
    double B = sc.nextDouble();
    double C = sc.nextDouble();
    double triangulo = (A * C) / 2;
    double circulo = 3.14159 * (Math.pow(C, 2));
    double trapezio = ((A + B) * C) / 2;
    double quadrado = Math.pow(B, 2);
    double retangulo = A * B;

    System.out.println("TRIANGULO: " + String.format("%.3f", triangulo));
    System.out.println("CIRCULO: " + String.format("%.3f", circulo));
    System.out.println("TRAPEZIO: " + String.format("%.3f", trapezio));
    System.out.println("QUADRADO: " + String.format("%.3f", quadrado));
    System.out.println("RETANGULO: " + String.format("%.3f", retangulo));
    
    sc.close();
  }
}

File: iniciante/swift/b1012.swift
import Foundation

let values = readLine()!.split(separator: " ")
let a = Double(values[0])!
let b = Double(values[1])!
let c = Double(values[2])!
    
var triangulo = (a * c) / 2
var circulo = 3.14159 * pow(c, 2)
var trapezio = ((a + b) * c) / 2
var quadrado = b * b
var retangulo = a * b

print(String(format: "TRIANGULO: %.3f", triangulo))
print(String(format: "CIRCULO: %.3f", circulo))
print(String(format: "TRAPEZIO: %.3f", trapezio))
print(String(format: "QUADRADO: %.3f", quadrado))
print(String(format: "RETANGULO: %.3f", retangulo))